### PR TITLE
Added example responses to Block Storage, Servers, Networks, and Files product pages.

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

**Comment by kenperkins** [comment]
@DonNateR is this a WIP or is it ready for review?

**Comment by DonNateR** [comment]
@kenperkins it's ready for review. Reviewing the page would actually be helpful, so that I can take any notes you may have and apply those notes to the other pages that need examples.

File: docs/services/blockstorage.rst [hunk 1]
```diff
@@ -1,89 +1,202 @@
...
+| `--volume-type` | Allows you to enter a volume type.   |
++-----------------+--------------------------------------+
+| `--rename`      | Allows you to rename a volume.       |
++-----------------+--------------------------------------+
+
+.. note::
+
+    *Command flags* will work with some commands and not with others. If you want
+    to know what *flags* will work with the specific command, type
+    `rack block-storage <action> --help`.
```
Comment (kenperkins) [inline docs/services/blockstorage.rst]: This wont actually show command flags, as you need a command and a resource, i.e. `rack block-storage volume list --help`. We should clean this language up.

File: docs/services/blockstorage.rst [hunk 2]
```diff
@@ -1,89 +1,202 @@
...
+| `--name`        | Allows you to name the volume or     |
+|                 | snapshot.                            |
++-----------------+--------------------------------------+
+| `--description` | Allows you to leave a description    |
+|                 | of the volume or snapshot.           |
++-----------------+--------------------------------------+
+| `--volume-type` | Allows you to enter a volume type.   |
++-----------------+--------------------------------------+
+| `--rename`      | Allows you to rename a volume.       |
++-----------------+--------------------------------------+
```
Comment (kenperkins) [inline docs/services/blockstorage.rst]: I worry about tables like this being interpreted as complete; it's neither complete nor accurate; it depends on the resource type you're using.

**Comment by kenperkins** [comment]
I think is

**Comment by kenperkins** [comment]
I think if we're going to show sample responses, we should show the command that would generate that response.

File: docs/services/blockstorage.rst (hunk 2 above)
Comment (annegentle) [inline docs/services/blockstorage.rst]: Let's not add tables anywhere. Please use bulleted lists when there's just a two-column table.

Comment (DonNateR) [inline docs/services/blockstorage.rst]: If this is the case, should we add a bulleted list under each action with their respective flags?

**Comment by DonNateR** [comment]
@kenperkins or @jrperritt: This PR is ready to be reviewed again.

**Comment by DonNateR** [comment]
Updated this PR with example responses for the Files page. However, there are conflicts which need to be resolved before merging and I am not sure what they are. I have saved the edited files locally if it's something that can't be resolved.

**Comment by DonNateR** [comment]
Closing this pull request and creating a new one with the edited files.

**Pull request closed without merging**